sambacc/addc.py

```diff
@@ -89,12 +93,21 @@ def add_group_members(group_name: str, members: list[str]) -> None:
     subprocess.check_call(cmd)
 
 
+def _filter_opts(
+    options: typing.Optional[typing.Iterable[tuple[str, str]]]
+) -> list[tuple[str, str]]:
+    _skip_keys = ["netbios name"]
```

Comment:
I hope this is because we already accept **netbios name** as `instance_name` in the config.

Reply:
That's correct.